src/test/java/com/audiogalaxy/audiogalaxy/PlaylistServiceTest.java

```diff
@@ -138,4 +141,33 @@ public void testGetSongsByPlaylistIdUnsuccessfully() {
 
         Assertions.assertThrows(InformationNotFoundException.class, () -> playlistService.getSongByPlaylistId(2l));
     }
+
+    @Test
+    @DisplayName("return playlist that was deleted as informational")
+    public void testDeletePlaylistSuccessfully() {
+        User currentlyLoggedInUser = new User("jeff", "jeff@gmail.com", "password");
+        when(userContext.getCurrentLoggedInUser()).thenReturn(currentlyLoggedInUser);
+
+        Playlist playlist = new Playlist("jazz music", "jazzy description");
+        playlist.getSongs().add(new Song("Jazz For The Quiet Times", "Smooth Jazz"));
+
+        when(playlistRepository.findByIdAndUserId(anyLong(), anyLong())).thenReturn(Optional.of(playlist));
+
+        Assertions.assertThrows(Exception.class, () -> playlistService.deletePlaylistId(playlist.getId()));
```

Comment:
why assert an exception when deleting the playlist successfully

Reply:
Let me take another look.  I was looking to send HTTP status ok; but, could not find one to work.